Dockerfile.openshift

```
@@ -1,5 +1,5 @@
 # This dockerfile is specific to building Multus for OpenShift
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.25-openshift-4.22 AS rhel9
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.26-openshift-5.0 AS rhel9
 ADD . /usr/src/multus-cni
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)